digitalearthau/config/eo3/products/ard_ls5.yaml

```diff
@@ -0,0 +1,177 @@
+name: ga_ls5t_ard_3
+description: Landsat 5 TM ARD, GA Collection 3
+metadata_type: eo3_landsat_ard
+
+metadata:
+    product:
+        name: ga_ls5t_ard_3
+
+measurements:
+    # NBAR
+-   name: nbar_blue
+    aliases:
+    -   nbar_band01
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbar_green
+    aliases:
+    -   nbar_band02
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbar_red
+    aliases:
+    -   nbar_band03
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbar_nir
+    aliases:
+    -   nbar_band04
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbar_swir_1
+    aliases:
+    -   nbar_band05
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbar_swir_2
+    aliases:
+    -   nbar_band07
+    dtype: int16
+    nodata: -999
+    units: '1'
+
+    # NBART
+-   name: nbart_blue
+    aliases:
+    -   nbart_band01
+    -   blue
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbart_green
+    aliases:
+    -   nbart_band02
+    -   green
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbart_red
+    aliases:
+    -   nbart_band03
+    -   red
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbart_nir
+    aliases:
+    -   nbart_band04
+    -   nir
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbart_swir_1
+    aliases:
+    -   nbart_band05
+    -   swir_1
+    dtype: int16
+    nodata: -999
+    units: '1'
+-   name: nbart_swir_2
+    aliases:
+    -   nbart_band07
+    -   swir_2
+    dtype: int16
+    nodata: -999
+    units: '1'
+
+    # Observation Attributes
+-   name: oa_fmask
+    aliases:
+    -   fmask
+    dtype: uint8
+    nodata: 0
+    units: '1'
+    flags_definition:
+        fmask:
+            bits: [0, 1, 2, 3, 4, 5, 6, 7]
+            description: Fmask
+            values:
+                '0': nodata
+                '1': valid
+                '2': cloud
+                '3': shadow
+                '4': snow
+                '5': water
+-   name: oa_nbar_contiguity
+    dtype: uint8
+    nodata: 255
+    units: '1'
+    flags_definition:
+        contiguous:
+            bits: [0]
+            values:
+                '1': true
+                '0': false
+-   name: oa_nbart_contiguity
+    dtype: uint8
+    nodata: 255
+    units: '1'
+    flags_definition:
+        contiguous:
+            bits: [0]
+            values:
+                '1': true
+                '0': false
+-   name: oa_azimuthal_exiting
+    dtype: float32
+    nodata: .nan
+    units: '1'
+-   name: oa_azimuthal_incident
+    dtype: float32
+    nodata: .nan
+    units: '1'
+-   name: oa_combined_terrain_shadow
+    dtype: uint8
+    nodata: -999
```

Comment:
this one needs to be fixed as raised by @simonaoliver in other products too

Reply:
:+1: